.github/pull_request_template.md

```diff
@@ -1,15 +1,18 @@
-**Which issue is resolved by this Pull Request:**
+<!-- Which issue is related to this Pull Request? -->
+<!-- Write "Resolves #<issue-number>" to automatically close the issue when PR merged. -->
 Resolves #
-
-<!-- You can delete this section if no corresponding issue -->
+<!--
+or if the PR is only a part of an issue, write the following instead:
+Part of #<issue-number>
+-->
 
 **Description of your changes:**
 
 
 **Checklist:**
-- Do you want this PR cherry picked to release branch?
+- [ ] Do you want this PR cherry picked to release branch?
 
-    If yes, please either
+  If yes, please either
     * ask the approver to add label `cherrypick-approved`, so that release manager
     will handle it in batch
     * or create a cherry pick PR to the release branch after this PR is merged
```

Comment:
wondering whether default one and less options. prefer PR author do it to reduce release manager's work and the possible merge conflict after weeks.

Reply:
we discussed in the design review, the pain is also we are PR authors too and batching cherry picks save us time in total (it saves context-switch time greatly, because cherry picking a PR requires a lot of actions after waiting).
(for rc2 I cherry picked around 10 PRs in 10 minutes without a single merge conflict, if release manager see a merge conflict, they can ask the PR author to create a PR to release branch instead.)

so if we want to suggest one, I'd suggest adding the label

Reply:
updated recommendation

Reply:
+1 to add "recommended". I assume it's also default path for non-Googler contributors.